raiden_contracts/data/source/services/ServiceRegistry.sol

```diff
@@ -220,6 +221,13 @@ contract ServiceRegistry is Utils, ServiceRegistryConfigurableParameters {
     event RegisteredService(address indexed service, uint256 valid_till, uint256 deposit_amount, Deposit deposit_contract);
 
     // @param _token_for_registration The address of the ERC20 token contract that services use for registration fees
+    // @param _controller The address that can change parameters and deprecate the ServiceRegistry
+    // @param _initial_price The amount of tokens needed iniailly for a slot
+    // @param _price_bump_numerator The ratio of price bump after deposit is made (numerator)
+    // @param _price_bump_denominator The ratio of price bump after deposit is made (denominator)
+    // @param _decay_constant The number of seconds after which the price becomes roughly 1/e.
+    // @param _min_price The minimum amount of tokens needed for a slot.
+    // @param _registration_duration The number of seconds (roughly) of a slot gained for a successful deposit.
```

Comment:
Is this roughly defined somewhere?

Reply:
I'll change "roughly" into "roughly, barring block time & miners' timestamp errors".

Reply:
Done.